src/components/Recipe.js

```diff
@@ -1,13 +1,26 @@
+import { nanoid } from 'nanoid';
 //import parse from 'html-react-parser';
```

Comment:
If you don't need that import I would suggest to delete it.

Reply:
I might need it later, cause I dont want to use dangerouslySetInnerHtml.  but for know its to complicated to use,because storybook and parse dont function together